testing/tests/DevExpress.exporter/jspdfParts/jspdf.printInfoCalculator.tests.js

```diff
@@ -0,0 +1,324 @@
+import { ExportPrintInfoCalculator } from 'exporter/jspdf/export_print_info_calculator.js';
+import { JSPdfPrintInfoCalculatorTestHelper } from './jspdfTestHelper.js';
+
+let helper;
+
+const JSPdfPrintInfoCalculatorTests = {
+    runTests(moduleConfig) {
+        helper = new JSPdfPrintInfoCalculatorTestHelper(this.jsPDFDocument);
+
+        QUnit.module('Print Info Calculator', moduleConfig, () => {
+            QUnit.test('Empty grid', function(assert) {
+                const done = assert.async();
+
+                const options = {
+                    jsPDFDocument: this.jsPDFDocument,
+                    cells: []
+                };
+                const calculatedOptions = ExportPrintInfoCalculator.calculate(options);
+                assert.strictEqual(calculatedOptions.columnStyles[0], undefined, 'columnStyles[0]');
+
+                done();
+            });
+
+            QUnit.test('1 column, 1 rows', function(assert) {
+                const done = assert.async();
```

Comment:
IHMO we should not use 'async()' just to make test safe if it doesn't wait for a promise.